### PR TITLE
Change type of field "value" in table `#_fields_values` from text to mediumtext

### PR DESCRIPTION
**Pull request by sergeytolkachyov**
Change type of field "value" in table `#_fields_values` from text to mediumtext for MySQL ONLY. PostgreSQL have only text field type with unlimited length. Pull Request for Issue #36065

### CODE REVIEW

**Comment by sergeytolkachyov** [comment]
Because J5.0.2 now is RC I moved it to 5.0.3.

**Comment by sergeytolkachyov** [comment]
@richard67 @HLeithner

**Comment by richard67** [comment]
@sergeytolkachyov Meanwhile 5.0.x is in stable phase, so I think this PR here should be made for the 5.1-dev branch. Could you rebase your PR or redo it for 5.1-dev?

File: installation/sql/postgresql/supports.sql [hunk 1]
```diff
@@ -226,7 +226,7 @@ CREATE TABLE IF NOT EXISTS "#__fields_values" (
 "field_id" bigint DEFAULT 0 NOT NULL,
 "item_id" varchar(255) DEFAULT '' NOT NULL,
 "value" text
-);
+	);
```
Comment (richard67) [inline installation/sql/postgresql/supports.sql]:
```suggestion
);
```
Please revert this change.

**Comment by sergeytolkachyov** [comment]
> @sergeytolkachyov Meanwhile 5.0.x is in stable phase, so I think this PR here should be made for the 5.1-dev branch. Could you rebase your PR or redo it for 5.1-dev?

Is this such a big change that it should only be in the minor version?

**Comment by richard67** [comment]
> Is this such a big change that it should only be in the minor version?

@sergeytolkachyov I will ask other maintainers or release managers for their opinion. Stay tuned.

**Comment by HLeithner** [comment]
> > @sergeytolkachyov Meanwhile 5.0.x is in stable phase, so I think this PR here should be made for the 5.1-dev branch. Could you rebase your PR or redo it for 5.1-dev?
> 
> Is this such a big change that it should only be in the minor version?

it's a feature and not a bugfix that's the reason why it should go into 5.1.

**Comment by sergeytolkachyov** [comment]
There is really no big difference when it will be enabled: in 5.0.3 or in 5.1. I just want to understand the logic.

**Comment by HLeithner** [comment]
everything which is not a bug is a feature, if you follow this you simple reduce the pain to upgrade. security stuff is another topic and a change like this doesn't need to hurry since you can fix it for your sites your self.

File: administrator/components/com_admin/sql/updates/mysql/5.0.3-2024-01-04.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+ALTER TABLE `#__fields_values` MODIFY `value` MEDIUMTEXT;
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/mysql/5.0.3-2024-01-04.sql:1]: Please rename the script from "5.0.3-2024-01-04.sql" to "5.1.0-2024-01-04.sql".

**Comment by sergeytolkachyov** [comment]
> > > @sergeytolkachyov Meanwhile 5.0.x is in stable phase, so I think this PR here should be made for the 5.1-dev branch. Could you rebase your PR or redo it for 5.1-dev?
> > 
> > 
> > Is this such a big change that it should only be in the minor version?
> 
> it's a feature and not a bugfix that's the reason why it should go into 5.1.

It seems to have done a rebase...

**Comment by richard67** [comment]
> It seems to have done a rebase...

@sergeytolkachyov Yes. Now the PR shows more changes than only yours, and there are conflicts, but I think that should be solved with the next upmerge from 5.0-dev to 5.1-dev by the release managers and then updating your branch to the 5.1-dev.

In addition it needs to rename the update SQL script, see my review comment.

So maybe it's easier if you just close this PR and make a new one for the 5.1-dev branch.

**Comment by richard67** [comment]
Thanks.
